app/domain/prompt.js

```diff
@@ -2,13 +2,14 @@
  * Generates system prompt
  * @returns {string}
  */
-const getPrompt = (summariesMode) => {
+const getPrompt = (summariesMode, items) => {
   const promptText = `
   You are a Gov UK DEFRA AI Assistant, whose job it is to retrieve and summarise information regarding available grants for farmers and land agents. 
   Documents will be provided to you with two constituent parts; an identifier and the content. The identifier will be at the start of the document, within a set of parentheses in the following format:
     (Title: Document Title | Grant Scheme Name: Grant Scheme the grant option belongs to | Source: Document Source URL | Chunk Number: The chunk number for a given parent document)
-    The start of the content will follow the "===" string in the document.
-    ${summariesMode && 'The documents provided may not include the relevant information to answer the query sufficiently. If the document do not mention or include the information,always respond only with "Unknown" as the answer.'}
+  The start of the content will follow the "===" string in the document.
+  ${summariesMode && 'The documents provided may not include the relevant information to answer the query sufficiently. If the document do not mention or include the information,always respond only with "Unknown" as the answer.'}
+  ${items && items.length > 0 && `Prioritise checking the following grant options: ${items.map((item) => item.title).join(', ')}`}
```

Comment:
Again adding in the items manually in the prompt doesn't need to be done as they are already taken from the vector search here https://github.com/DEFRA/fcp-find-ai-frontend/blob/5385b5b692421a154e5f2f312d239df72fd33526/app/services/query-service.js#L97C11-L97C20